main.tf

```diff
@@ -220,6 +220,34 @@ module "aws_deploy-main_mon-eu-north-1" {
   }
 }
 
+module "aws_deploy-main_backup-eu-central-1" {
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.3.1"
```

Comment:
Once https://github.com/aeternity/terraform-aws-aenode-deploy/commit/542cf48efabf646321186aa059dbc7692e0470ca is release in v2.4, bump the module version

Reply:
7b0b985